frontend-thea-aa/src/lib/components/AirQualityRanking.svelte

```diff
@@ -7,25 +7,26 @@
 	let amountToShow = $state(3)
 </script>
 
-<h3>Air Quality Ranking</h3>
-
-<table>
-	<thead>
-		<tr>
-			<th>Rank</th>
-			<th>City</th>
-			<th>Quality</th>
-		</tr>
-	</thead>
-	<tbody>
-		{#each props.items.slice(0, amountToShow) as item, rank}
-			<tr>
-				<td>{rank}</td>
-				<td>{item.station}</td>
-				<td>{item.airQuality}</td>
-			</tr>
-		{/each}
-	</tbody>
+<div class="max-w-4xl mx-auto py-6">
+<h3  class="text-l font-bold mb-4 text-gray-800">Air Quality Ranking</h3>
```

Comment:
https://www.shadcn-svelte.com/docs/theming
instead of the specific colors, use these color classes everywhere so that when light and dark theme are changed, it works nicely